harness/buildString.js

```diff
@@ -14,3 +14,19 @@ function buildString({ loneCodePoints, ranges }) {
   }
   return result;
 }
+
+function test(regex, string, expression) {
```

Comment:
Any suggestions for a better name?

Also, should we rename `buildString.js` to something like `regExpUtils.js`? (Again, suggestions welcome.)